source/fundamentals/linq.txt

```diff
@@ -497,6 +497,82 @@ The following is a subset of the returned results:
        ]
     }
 
+$vectorSearch
```

Comment:
This part looks good. 
VectorSearch is not only limited to IMongoQueryable. We also have [builders](https://github.com/mongodb/mongo-csharp-driver/blob/master/tests/MongoDB.Driver.Tests/PipelineDefinitionBuilderTests.cs#L411C11-L411C12) and [Fluent API](https://github.com/mongodb/mongo-csharp-driver/blob/master/tests/MongoDB.Driver.Tests/Search/AtlasSearchTests.cs#L577) options. I think it's worth documenting all options.

Also should we have a separate page for `VectorSearch` (like we do for `AtlasSearch`) to increase visibility? cc @rishitb-mongodb?

Reply:
I was curious about this point and would like to hear Rishit's PoV – I originally didn't add documentation to the Builders page we have since it didn't seem that there was a specific builder for the VectorSearch stage (i.e. nothing like a  

I'd also want to be wary of duplicating information – there is ample documentation on Vector Search in Atlas already so I'd prefer to link to that when able, unless there are C# driver-specific quirks that are worth calling out.

Reply:
I guess I'm struggling with the "natural" point to describe Vector Search in our docs since it seems like neither Builders nor LINQ is fully appropriate. Maybe a section in the Aggregations page with a link to the detailed documentation?

Reply:
I think we need to have some sort of consistency, either within our documentation (by having a dedicated page for Atlas Vector Search just like we do for Atlas Search) OR with other driver's documentation (which is what the above approach does by mimicing what the Java driver was doing for Atlas Vector Search). I know there is a standardization effort coming soon for this. I didn't know that we have Builders and FluentAPI option for Vector Search as well. If this is the case then I agree this page shouldn't be under LINQ.  If a developer can do Vector Search using Builders, Fluent API or LINQ then we should have a dedicated page for it but we don't need to repeat everything in the main Atlas documentation, just redirect to it wherever needed.